code/__HELPERS/lists.dm

```
@@ -563,16 +563,20 @@ proc/dd_sortedtextlist(list/incoming, case_sensitive = 0)
 	return sorted_text
 
 
-proc/dd_sortedTextList(list/incoming)
+/proc/dd_sortedTextList(list/incoming)
 	var/case_sensitive = 1
 	return dd_sortedtextlist(incoming, case_sensitive)
 
+/proc/subtypesof(var/path) //Returns a list containing all subtypes of the given path, but not the given path itself.
+	if(!path || !ispath(path))
+		return

```

You do **not** want this to fail silently. `CRASH` here.